custom_dc/build_image.md

```diff
@@ -41,82 +41,79 @@ If you want to pick up the latest prebuilt version, do the following:
 
 > **Note:** If you are working on a large-scale customization, we recommend that you use a version control system to manage your code. We provide procedures for Github, and assume the following:
 - You have a Github account and project.
-- You have created a fork off the base Data Commons `website` repo (https://github.com/datacommonsorg/website){: target="_blank"} and a remote that points to it, and that you will push to that fork. 
-
+- You have created a fork off the base Data Commons `website` repo [https://github.com/datacommonsorg/website](https://github.com/datacommonsorg/website){: target="_blank"}. 
 
 ### Sync a local workspace to the stable release
 
-If you are using a version control system other than Github, you can download a ZIP or TAR file from [https://github.com/datacommonsorg/website/releases/tag/customdc_stable](https://github.com/datacommonsorg/website/releases/tag/customdc_stable){: target="_blank"}. 
-
-In Github, use the following procedure.
+#### Clone the stable branch only
 
-1. If you want to reuse the root directory you previously created and cloned, skip to step 3. 
-If you want to create a new source directory and start from scratch, clone the repo up to the stable release tag:
+Use this procedure if you are not using Github, or if you are using Github and want to create a new source directory and start from scratch. If you are using Github and want to use the same directory and files you previously cloned, skip to the next section.
 
+1. Run the following command:
    <pre>
-   git clone https://github.com/datacommonsorg/website --branch customdc_stable --single-branch  [<var>DIRECTORY</var>]
+   git clone https://github.com/datacommonsorg/website --b customdc_stable --single-branch  [<var>DIRECTORY</var>]
    </pre>
-1. Change to the root directory:
-
+   This creates a new local branch called `customdc_stable` set to track the Data Commons repo branch. 
+1. To verify, run:
    <pre>
    cd website | cd <var>DIRECTORY</var>
+   git branch -vv
    </pre>
+   You should see output like the following:
 
-1. Create a new branch synced to the stable release:
+   ```
+   * customdc_stable 83732891 [origin/customdc_stable] 2024-11-06 Custom DC stable release (#4710)
+   ```
+   Rather than developing on this default branch, we recommend that you create another branch, as described in the next step.
+
+#### Create a remote and sync a local branch to the stable branch
+This procedure assumes that you are using Github and have already created a forked repo of `datacommonsorg/website`.
 
+1. From the `website` or other directory where you have cloned the code, create a remote that points to your fork, and only uses the stable branch:
    <pre>
-   git checkout -b <var>BRANCH_NAME</var> customdc_stable
+   git remote add -t customdc_stable <var>REMOTE_NAME</var> <var>FORK_URL</var>
```

Comment:
I think we should probably have this track `customdc_stable` on the main repo rather than the fork, so the user doesn't have to keep their fork in sync. WDYT?

Reply:
You're right, that makes more sense! So in that case, the fork is actually kind of irrelevant here, because they will only push to it, rather than pull from it, right?